packages/b/buckets.json

```
@@ -24,13 +24,13 @@
     "map"
   ],
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/mauriciosantos/Buckets-JS.git",
+    "source": "npm",
+    "target": "buckets-js",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "dist",
         "files": [
-          "buckets*.js"
+          "**/*"
```

```suggestion
          "buckets?(.min).js?(.map)"
```
`**/*` is potentially dangerous as it leaves us open to abuse as we'll copy anything published, let's use a more specific glob for the files we need, based on the existing one. [Glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=buckets%3F%28.min%29.js%3F%28.map%29&matches=false&tests=buckets.js&tests=buckets.min.js&tests=buckets.min.js.map)